### PR TITLE
Minor changes to console api docs

### PR DESCRIPTION
**Pull request by evandiewald**
In the Delete Label call, device_id is a path parameter, not a body parameter. 
ln Add Device to Label, changed 'label' parameter to 'label_id' for consistency.

### CODE REVIEW

**Comment by evandiewald** [comment]
Realized that the second change is incorrect. The body parameter is in fact 'label', not 'label_id'

**Comment by PharkMillups** [comment]
Thanks, Evan! @madninja Can you take a quick look?

File: docs/api/console.mdx [hunk 1]
```diff
@@ -474,7 +474,7 @@ Delete a device label, along with all of its device links.
   values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
 <TabItem value="request">
 
-_Body Parameters_
+_Path Parameters_
```
Comment (samgutentag) [inline docs/api/console.mdx:477]: looks good

File: docs/api/console.mdx [hunk 2]
```diff
@@ -569,11 +569,7 @@ _Path Parameters_
 200: OK
 
 ```json
-{
-  "id": "868ea783-f252-4fc4-b724-a155680c0bc9",
-  "multi_buy": 1,
-  "name": "Blue Label"
-}
+Label deleted
```
Comment (samgutentag) [inline docs/api/console.mdx:572]: this is in the wrong spot,

**Comment by samgutentag** [comment]
corrected mixup in `Search for label by name.` and `Delete a device label`

**Review by samgutentag — DISMISSED** [review]

**Review by samgutentag — APPROVED** [review]

**Pull request merged**